_posts/2019-08-11-pre-post-events-sustainable-train-travels.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "Pre-/Post Events: sustainable 🚄 train travels from and to RustFest Barcelona"
+authors:
+  - dns2utf8
+socialImageSrc: "/assets/posts/barcelona/train-map.png"
+---
+
+## The big plan 🚆
+
+What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
+We want to make that happen and are organizing a community travel from the north of Europe all the way down to Barcelona, with stops in between.
+We want to encourage more people to join us or plan similar travel events.
+
+So what do we need for that?
+
+1. Is there a path to go by train?
+2. How far can you go in one day?
+3. Which stops are interesting?
+4. Where can I meet other people?
+
+1 and 2 you have to answer for yourself.
+
+3 and 4 we can help. We are [collecting all pre-/post events](https://barcelona.rustfest.eu/pre_post_events/) on our website.
+
+Beeing able to talk to your peers directly makes planning easy.
+The [Rust User Forum](https://users.rust-lang.org/c/community) is a great place to organize.
+
+### Tickets or Interrail
+
+Usually people buy tickets from A to B, sometimes by locking in on a single connection.
+However, a travel of this length has some challenges.
+
+When traveling huge distances over many countries sometimes delays happen or a train might break down.
+With a regular ticket you have to find staff and organize a reroute on the spot.
+
+This is where Interrail saves the day.
+With the Interrail global pass you buy travel days over a period of time.
+So you can simply catch the next train and continue your journey.
+
+If you buy 7 travel days within one month, you have the flexibility to ride any train outside of your home country on 7 days.
```

Comment:
```suggestion
+If you buy the "7 travel days within one month" ticket, you'll have the flexibility to take any train outside of your home country for 7 days. You can take any train outside of your home country for 7 days.
+Included in this ticket, you're allowed *one outbound journey from* your country* and *one inbound journey to your country*. You can find more information [here](https://www.interrail.eu/en/interrail-passes/what-is-interrail/how-use-your-interrail-pass) on how an Interrail train ticket work.
```

Reply:
@dns2utf8 see if you like how I have reworded

Reply:
Will refine it a bit more